gc/base/standard/ParallelGlobalGC.hpp

```diff
@@ -334,6 +334,10 @@ class MM_ParallelGlobalGC : public MM_GlobalCollector
 
 	virtual void workStackOverflow() {}
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	virtual bool reinitializeForRestore(MM_EnvironmentBase *env);
```

Comment:
do we require a function description for `reinitializeForRestore`? or is there an existing one?

Reply:
Yup! Just added one to the base class `gc/base/GlobalCollector.hpp` function which is being implemented here